components/clp-package-utils/clp_package_utils/scripts/native/search.py

```diff
@@ -220,6 +230,11 @@ def main(argv):
         help="Perform the query and count the number of results.",
```

Comment:
```suggestion
        help="Count the number of results.",
```
Sorry, I think you had something like this the previous PR and I thought it was unclear. But in light of the new option, I think it's simpler.